website_sale_google_shopping/README.md

```diff
@@ -0,0 +1,242 @@
+[![License: AGPL-3](https://img.shields.io/badge/licence-AGPL--3-blue.svg)](http://www.gnu.org/licenses/agpl-3.0-standalone.html)
+
+============================
+Website Sale Google Shopping
+============================
+
+Description
+===========
+
+This module provides a feed of products from your e-commerce to allow Google Shopping index them and show as search results.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * git clone https://github.com/OCA/e-commerce.git --branch 8.0
+ * git clone https://github.com/OCA/product-attribute.git --branch 8.0
+ * make them available to odoo by adding their locations to the addons_path in
+   /etc/odoo-server.conf
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * no configuration required
+
+Usage
+=====
+
+Visit the feed page at http://yourdomain/google-shopping.xml
+
+To use this module succefully, you need to known some aspects:
+
+- Each kind of product has different requirements attributes. The requirements depend of specific google category and your target country.
+- Usual product with detailed product attributes are: clothes(color, material, and size), furniture(material, pattern, and color), electronic devices(color) and toy(group of ages).
+
+How to complete product attributes?
+
+- Sales, product sheet, sales tab, website field: Here, you will find all field related to google shopping feed.
```

Comment:
Sales, product sheets, sales tabs, website fields: Here, you will find all fields related to Google shopping feed.